COLLABORATORS.md

```diff
@@ -36,3 +39,22 @@ After installing `rimraf`, `yarn` command can be run normally when installing de
 ts-jest must match the major version of jest. Matching the minor version is preferred. Matching the version patch is not required.
 
 When merging PRs which fix bugs, it is preferable to increment the version patch version so that the changes can be published to NPM.
+
+## Tests
+
+All new features or bugfixes should be accompanied with a new test, to ensure that the change works as intended, and to make sure we don't inadvertently break it in the future through refactring
+
+If you are simply refactoring code, it is not needed to add a test.
+
+### How to add tests
+- Tests are added by creating a new file in tests/__tests__ 
+- A test usually consists of two parts
+	 - A directory under tests/ that have some files that should result in a specific output when jest is passed through it
+	 - A file under tests/__tests__ that uses the runJest command to run Jest on the specified directory, and makes asserts on the output of stdout and stderr
+	 	When asserting things on the output, it is common to both assert something about the exit code, and something about the output. Always assert on the output first, before asserting on the status code - as it leads to much nicer error messages when refactoring. (You get the full output text, rather than just '0 didn't equal 1')
+
+### How to run tests
+Tests are run through `yarn test`
+
+## Code Of Conduct
```

Comment:
I really don't want to introduce this. Having a code of conduct seems to be something of a magnet for extremists on all sides